testutil/integration/model_test.go

```diff
@@ -1,115 +1,217 @@
+/*Case Range
+Model Extract：
+	DO Not Extract（TensorRT，非模型（待定））；
+	Do Extract (SavedModel,ONNX,GraphDef,NetDef,Keras H5, CaffeModel, TorchScript, MXNetParams, PMML)-create ModelJob automatically
+Model Convert:
+	Do Not Convert
+	Do Convert (MXNetParams -> ONNX,Keras H5 -> SavedModel,CaffeModel -> NetDef)
+API
+	modeljob list, create, get, delete, get events
+*/
 package integration
 
 import (
-	"encoding/json"
-	"fmt"
-	"net/http"
-	"time"
+	modeljobsv1alpha1 "github.com/kleveross/klever-model-registry/pkg/apis/modeljob/v1alpha1"
+	"github.com/kleveross/ormb/pkg/oras"
+	"github.com/kleveross/ormb/pkg/ormb"
 
+	"fmt"
 	httpexpect "github.com/gavv/httpexpect/v2"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	modeljobsv1alpha1 "github.com/kleveross/klever-model-registry/pkg/apis/modeljob/v1alpha1"
-	"github.com/kleveross/klever-model-registry/pkg/registry/models"
+	"net/http"
+	"os"
 )
 
 var _ = Describe("Model Registry", func() {
-	const timeout = time.Second * 5
-	const interval = time.Second * 1
+
+	var modelExtract = []struct {
+		model   string
```

Comment:
Should we change all the lowercase to uppercase so that other packages can also use the struct?

Reply:
There is no need to capitalize since we only use these structs in e2e tests